port/unix/omrsock.c

```diff
@@ -79,6 +79,9 @@ get_os_family(int32_t omrFamily)
 static int32_t
 get_os_socktype(int32_t omrSockType)
 {
+	/* Mask sockType. It should be defined in this range. */
+	omrSockType &= 0x00FF;
```

Comment:
minor nit: do we want to just use `0xFF`? I'm not sure what the style of the existing code is (or if there is another reason for using this format)

Reply:
There is not much of a reason, I only included it for consistency with get_os_socket_flag function.
In `omrportsock.h` where I defined Socket Flags:
```
#define OMRSOCK_O_ASYNC 0x0100
#define OMRSOCK_O_NONBLOCK 0x1000
```
It modifies the first two hex could be modified as well to represent socket flag, which is passed in through socktype in omrsock_socket. In get_os_socket_flag, gets the first two hex 0xFF00.

Maybe we should keep it for being consistent with get_os_socket_flag but I could also remove it.